storage/log_storage.go

```diff
@@ -100,12 +100,6 @@ type LogTreeTX interface {
 	// employing this property. Consult the call sites of this method to be sure.
 	DequeueLeaves(ctx context.Context, limit int, cutoff time.Time) ([]*trillian.LogLeaf, error)
 
-	// AddSequencedLeaves stores the passed in leaves at the log positions
-	// specified in the `LeafIndex` field. The indices must be contiguous.
-	//
-	// See LogStorage.AddSequencedLeaves comment for more details.
-	AddSequencedLeaves(ctx context.Context, leaves []*trillian.LogLeaf, timestamp time.Time) ([]*trillian.QueuedLogLeaf, error)
```

Comment:
All the implementations have been cleaned up?

Reply:
MySQL?
In-memory storage impl? (not sure it still exists)
Mocks?

Reply:
The MySQL one is still used but only internally. Could you unexport it as `addSequencedLeaves`?

Reply:
No implementions use this now

Reply:
It looks like the MySQL one might still exist? Anyway it can be sorted out later.